From 0b13ba237363308b239a96ac902341fe2612614d Mon Sep 17 00:00:00 2001
From: Simon Dinkin <simonx.dinkin@intel.com>
Date: Thu, 15 Dec 2016 15:02:57 +0200
Subject: [PATCH] rrm: beacon request bug fix

fix mismatch in order of sending parameters from ctrl_iface
to rrm.
(num of repetitions parameter was swaped with operating class)

Signed-off-by: Simon Dinkin <simonx.dinkin@intel.com>
---
 hostapd/ctrl_iface.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hostapd/ctrl_iface.c b/hostapd/ctrl_iface.c
index a200390..768ef4c 100644
--- a/hostapd/ctrl_iface.c
+++ b/hostapd/ctrl_iface.c
@@ -2671,8 +2671,8 @@ static int hostapd_ctrl_iface_req_beacon(struct hostapd_data *hapd, char *cmd,
 	}
 
 exit:
-	ret_dialog_token = hostapd_send_beacon_req(hapd, addr, op_class, channel,
-			num_of_repetitions, measurement_request_mode, random_interval,
+	ret_dialog_token = hostapd_send_beacon_req(hapd, addr, num_of_repetitions,
+			measurement_request_mode, op_class, channel, random_interval,
 			measurement_duration, mode, bssid, ssid_p, rep_cond_p,
 			rep_cond_threshold_p,rep_detail_p, ap_ch_rep_p, ap_ch_rep_len,
 			req_elem_p, req_elem_len, channel_width_p,
-- 
2.10.1

